Framework/Algorithms/src/Qxy.cpp

```diff
@@ -378,6 +379,21 @@ std::vector<double> Qxy::logBinning(double min, double max, int num) {
   return outBins;
 }
 
+double
+Qxy::getQminFromWs(const API::MatrixWorkspace_const_sptr &inputWorkspace) {
```

Comment:
If the pointer is never NULL, change the signature to `const API::MatrixWorkspace &`